reconcile/change_owners/change_log_tracking.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from reconcile.change_owners.bundle import (
+    NoOpFileDiffResolver,
+    QontractServerDiff,
+)
+from reconcile.change_owners.change_owners import fetch_change_type_processors
+from reconcile.change_owners.change_types import ChangeTypeContext
+from reconcile.change_owners.changes import aggregate_file_moves, parse_bundle_changes
+from reconcile.utils import gql
+from reconcile.utils.runtime.integration import NoParams, QontractReconcileIntegration
+from reconcile.utils.semver_helper import make_semver
+from reconcile.utils.state import init_state
+
+
+class ChangeLogIntegration(QontractReconcileIntegration[NoParams]):
+    def __init__(self) -> None:
+        super().__init__(NoParams())
+        self.qontract_integration = "change-log-tracking"
+        self.qontract_integration_version = make_semver(0, 1, 0)
+
+    @property
+    def name(self) -> str:
+        return self.qontract_integration
+
+    def run(self, dry_run: bool) -> None:
+        change_type_processors = [
+            ctp
+            for ctp in fetch_change_type_processors(
+                gql.get_api(), NoOpFileDiffResolver()
+            )
+            if ctp.labels and "change_log_tracking" in ctp.labels
+        ]
+        state = init_state(
+            integration=self.name,
+        )
+        int_state_path = state.state_path
+        state.state_path = "bundle-archive/diff"
```

Comment:
mutate `state_path` here then reset back is not very clean, can we allow `init_state` to accept `state_path` to override on `State.__init__`, then we can have one for integration state, another for `diff` state.

Reply:
i took a different direction in https://github.com/app-sre/qontract-reconcile/pull/4620/commits/1c86071e96c02a496a336dc7fb9ecb8cae52fc60

Reply:
actually used 2 states: https://github.com/app-sre/qontract-reconcile/pull/4620/commits/d33d772d24be985a0b2928c3cdf3dbf4cc851271